src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -5103,6 +5119,15 @@ private void OnGrabHandleColorChanged(float r, float g, float b, float a)
         {
             GrabHandleColor = new Color(r, g, b, a);
         }
+        public void SetFontVariation(string axis, float value)
+        {
+            int index = RegisterFontVariationProperty(axis);
+            Object.SetProperty(SwigCPtr, index, new PropertyValue(value));
```

Comment:
```suggestion
            Object.InternalSetPropertyFloat(SwigCPtr, index, value);
```
NUI 레벨에서 PropertyValue 를 생성하지 않고 DALi로 값을 넘기는 편이 나을 것 같습니다.

Reply:
Done